akka-http-core/src/main/scala/akka/http/impl/engine/HttpConnectionTimeoutException.scala

```diff
@@ -0,0 +1,5 @@
+package akka.http.impl.engine
+
+import scala.util.control.NoStackTrace
+
+class HttpConnectionTimeoutException(msg: String) extends RuntimeException(msg) with NoStackTrace
```

Comment:
Hm okey; may be a bit difficult to apply retroactively but we can do so from now on I guess.